mesonbuild/compilers/cpp.py

```diff
@@ -221,6 +231,9 @@ def get_option_compile_args(self, options):
         std = options['cpp_eh']
         if std.value != 'none':
             args.append('/EH' + std.value)
+        std = options['cpp_std']
+        if std.value not in ['none', 'c++11']:
+            args.append('/std:' + std.value)
```

Comment:
Why isn't this value added for `c++11`? Please add a comment about that.

Also, why aren't we warning when `cpp_std:` on an MSVC compiler that does not support the `/std:` argument?